### PR TITLE
[remediations] Update visual side of resolution selection.

### PR DESCRIPTION
**Pull request by Hyperkid123**
jira: https://issues.redhat.com/browse/RHCLOUD-8253

### TODO
- [x] update visuals
- [x] review step selection
- [x] ~~correct recommended label condition~~
- [x] correct vulnerabilities attribute format

note that notification has to be emitted by the actual application. The wizard returns prepared notification data. That's why its not visible in the recording.
wizard-rem

### CODE REVIEW

File: packages/remediations/src/steps/IssueResolutionStep.scss [hunk 1]
```diff
@@ -1,5 +1,33 @@
...
+        }
+        .pf-c-tile__body {
+            display: flex;
+            .pf-c-content {
+                margin-top: auto;
+            }
+        }
+    }
+    .ins-c-reboot-required {
+        color: #826110
```
Comment (PanSpagetka) [inline packages/remediations/src/steps/IssueResolutionStep.scss:25]: Is there some PF variable that matches this color?

File: packages/remediations/src/steps/IssueResolutionStep.scss [hunk 2]
```diff
@@ -1,5 +1,33 @@
...
 
-.ins-c-resolution-option .pf-c-check{
-    align-items: baseline;
+.ins-c-resolution-container {
+    display: flex;
+    flex-wrap: wrap;
 }
+.ins-c-resolution-option{
+    width: 50%;
+    padding: 8px;
```
Comment (PanSpagetka) [inline packages/remediations/src/steps/IssueResolutionStep.scss:9]: var(--pf-global--spacer--sm)

File: packages/remediations/src/steps/IssueResolutionStep.scss (hunk 1 above)
Comment (Hyperkid123) [reply packages/remediations/src/steps/IssueResolutionStep.scss:25]: I believe there is not.

Comment (Hyperkid123) [reply packages/remediations/src/steps/IssueResolutionStep.scss:25]: Wait. This might be it: --pf-global--warning-color--200

**Review by karelhala — APPROVED** [review]

**Comment by karelhala** [comment]
Let's wait for @katierik visual review.

**Review by katierik — APPROVED** [review]
Looks great!

**Pull request merged**